actions/nuxeo-action-element-styles.html

```diff
@@ -0,0 +1,51 @@
+<!--
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<link rel="import" href="../../polymer/polymer.html">
+
+<!--
+  Styles module to be used by action elements.
+
+  Custom property | Description | Default
+  ----------------|-------------|----------
+  `--nuxeo-action-element` | Mixin applied to the action element | { display: inline-block; }
+  `--nuxeo-action-element-show-label` | Mixin applied to the action element when showing label | {}
+  `--nuxeo-action-element-label` | Mixin applied to action label | {}
+-->
+<dom-module id="nuxeo-action-element-styles">
+  <template>
+    <style>
+      :host {
+        display: inline-block;
+        @apply --nuxeo-action-element;
+      }
+
+      :host([show-label]) {
```

Comment:
Since we don't yet have a need to expose this and the label mixin wdyt about keeping the style module but not exposing the mixin? we can add it back anytime.. keep in mind mixins are expanded to css variables at runtime....

Reply:
We're using the label mixing (`--nuxeo-action-element-label`) on the demo and web ui. And if people want to switch the icon to the right, for example, they most certainly will need `--nuxeo-action-element`, although I'm not sure how likely this will be...